user_data.sh

```diff
@@ -53,6 +49,12 @@ done
 chown $SSH_USER:$SSH_USER $KEYS_FILE
 chmod 600 $KEYS_FILE
 mv $TEMP_KEYS_FILE $KEYS_FILE
+
+cat <<"SSHCONFIG" > /home/${ssh_user}/.ssh/config
```

Comment:
Same. Move these lines outside of file `update_ssh_authorized_keys.sh` (after "EOF" on line 58, for example) and set correct permissions to `.ssh/config` file (whatever it should be).